go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/DataDog/test-infra-definitions
 
-go 1.22
+go 1.22.5
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Thank you!!